src/main/java/persistence/sql/mapping/Columns.java

```diff
@@ -0,0 +1,79 @@
+package persistence.sql.mapping;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Transient;
+import persistence.sql.ddl.exception.AnnotationMissingException;
+import persistence.sql.ddl.exception.IdAnnotationMissingException;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Columns implements Iterable<ColumnData> {
+
+    private final List<ColumnData> columns;
+
+    private Columns(List<ColumnData> columns) {
+        this.columns = columns;
+    }
+
+    @Override
+    public Iterator<ColumnData> iterator() {
+        return columns.iterator();
+    }
+
+    public static Columns createColumns(Class<?> clazz) {
+        checkIsEntity(clazz);
+        List<ColumnData> columns = Arrays.stream(clazz.getDeclaredFields())
+                .filter(field -> !field.isAnnotationPresent(Transient.class))
+                .map(ColumnData::createColumn)
+                .collect(Collectors.toList());
+
+        checkHasPrimaryKey(columns);
+        return new Columns(columns);
+    }
+
+    public static Columns createColumnsWithValue(Class<?> clazz, Object entity) {
+        List<ColumnData> columns = Arrays.stream(clazz.getDeclaredFields())
```

Comment:
```suggestion
    public static Columns createColumnsWithValue(Class<?> clazz, Object entity) {
        checkIsEntity(clazz);
        List<ColumnData> columns = Arrays.stream(clazz.getDeclaredFields())
```
위의 `createColumns` 처럼 `checkIsEntity` 를 호출해야 하지 않을까요? 👀